Haskell/MaîtreD.hs

```diff
@@ -1,11 +1,19 @@
 module MaîtreD where
 
-import Data.Time (ZonedTime)
+import Data.Time (ZonedTime (zonedTimeToLocalTime, zonedTimeZone))
```

Comment:
This may only be ignorance on my part, but what does it mean when the imports are double-bracketed?